src/course-updates/hooks.jsx

```diff
@@ -0,0 +1,114 @@
+import { useDispatch, useSelector } from 'react-redux';
+import moment from 'moment/moment';
+import { useEffect, useState } from 'react';
+import { useToggle } from '@edx/paragon';
+
+import { getCourseHandouts, getCourseUpdates } from './data/selectors';
+import { REQUEST_TYPES } from './constants';
+import { COMMA_SEPARATED_DATE_FORMAT } from '../constants';
```

Comment:
Can we follow a uniform approach of grouping local imports from nearest to the ballistic location?

```suggestion
import { COMMA_SEPARATED_DATE_FORMAT } from '../constants';
import { getCourseHandouts, getCourseUpdates } from './data/selectors';
import { REQUEST_TYPES } from './constants';
```

Reply:
Done